core/server/models/member.js

```diff
@@ -1,32 +1,23 @@
 const ghostBookshelf = require('./base');
-const security = require('../lib/security');
 
 const Member = ghostBookshelf.Model.extend({
     tableName: 'members',
 
-    onSaving() {
-        ghostBookshelf.Model.prototype.onSaving.apply(this, arguments);
-
-        if (this.hasChanged('password')) {
```

Comment:
Why is this bit going away in this changeset?

Reply:
Members no longer have passwords! This was missed when removing the password field from the database, but seeing as we're touching the models here - I cleaned it up :relaxed: